### ISSUE TITLE
Accessing google.visualization

### ISSUE THREAD

**Issue by jkon** [FERNman/angular-google-charts#14]
I was wondering how you suggest to add the google object to an angular project so it can be access for making formatters or some of the more advanced chart operations (currently trying to figure out how to do a bar diff chart)
Thanks

**Comment by jkon** [comment]
Sweet, thanks!

**Comment by jkon** [comment]
so, I see how you used the script loader service to get the chart packages, but what if I want to make a formatter? I still need the google.visualization object, right? Like in your formatter example:
``` myFormatter = [
  { formatter: new google.visualization.Dateformat({formatType: 'long'}), colIndex: 1 },
  { formatter: new google.visualization.Dateformat({formatType: 'long'}), colIndex: 3 }
]; ```

**Comment by jkon** [comment]
Having a new issue after updating to 0.0.7. All of my module.ngfactory.js files are throwing this error:
"export 'ɵa' (imported as 'i39') was not found in 'angular-google-charts'

**Comment by jkon** [comment]
i downgraded to 0.0.6 and had the same errors. had to go back to 0.0.5 to get rid of them

**Comment by FERNman** [comment]
Could you please provide explanation on how to reproduce this?

**Comment by jkon** [comment]
I tried to create a stackblitz to reproduce, but I can't seem to. It must be something specific to my project. I can add that when I try to render the chart, I get this error after every change detection: 
TypeError: ctor is not a constructor
at _createClass (core.js:10628)
    at _createProviderInstance$1 (core.js:10602)
    at resolveNgModuleDep (core.js:10553)
    at NgModuleRef_.get (core.js:11595)
    at resolveDep (core.js:12074)
    at createClass (core.js:11934)
    at createDirectiveInstance (core.js:11777)
    at createViewNodes (core.js:13239)
    at createEmbeddedView (core.js:13116)
    at callWithDebugContext (core.js:14524)

I'm not sure what's going on or why it is happening with the newer versions only

**Comment by jkon** [comment]
hm, I have been trying to create a stackblitz example that reproduces the errors I was seeing, but I just couldn't. I added all the modules my app imports and couldn't. So, I decided to try seeing if it was a package version issue with some other package, so I blew out my local node_modules and reinstalled everything and now the errors are gone from my project. Just one of those shruggie moments. Anyway, while building the stackblitz I also did an example of how to use the script loader service. Here is a snippet of how I used it to make a formatter that I think could help others:
```
constructor( private loaderService: ScriptLoaderService ) {}

  ngOnInit() {
    this.loaderService.onReady.subscribe( () => {
      this.loaderService.loadChartPackages(this.type).subscribe( () => {
        console.log('loaded chart packages');
        const formatter = new google.visualization.BarFormat();
        console.log(formatter);
      });
      
    });
  }
```

It's worth noting that this.type needs to be the translated string that your package returns with [GoogleChartPackagesHelper.getPackageForChartName(this.type)].  That helper isn't available, so I had to look at it to figure out what string to actually pass. That helper  being available would be nice.
Again, thanks for the quick turnaround on exposing the service
